web3/middleware/cache.py

```diff
@@ -195,7 +195,7 @@ def middleware(method: RPCEndpoint, params: Any) -> RPCResponse:
 
 
 _time_based_cache_middleware = construct_time_based_cache_middleware(
-    cache_class=functools.partial(lru.LRU, 256),
+    cache_class=functools.partial(lru.LRU, 256)(),
```

Comment:
@fselmo started seeing CI lint failures for this incorrect default argument. Fix here was surprising to me, I'm not totally sure why it began failing CI suddenly.